### PR TITLE
add boba l2 rinkeby and mainnet

### PR DESCRIPTION
**Pull request by InoMurko**
Hey, I'd like to add Boba Rinkeby and Boba mainnet chains as we work towards Gnosis Safe support!
https://github.com/ethereum-lists/chains/pull/528/files

### CODE REVIEW

File: gnosis/eth/clients/etherscan_client.py [hunk 1]
```diff
@@ -31,6 +31,8 @@ class EtherscanClient:
         EthereumNetwork.BINANCE: "https://bscscan.com",
         EthereumNetwork.MATIC: "https://polygonscan.com",
         EthereumNetwork.OPTIMISTIC: "https://optimistic.etherscan.io",
+        EthereumNetwork.BOBA_RINKEBY: "https://blockexplorer.rinkeby.boba.network/",
```
Comment (Uxio0) [inline gnosis/eth/clients/etherscan_client.py:34]: This is not etherscan, it should be removed from here

File: gnosis/eth/clients/etherscan_client.py [hunk 2]
```diff
@@ -42,6 +44,8 @@ class EtherscanClient:
         EthereumNetwork.BINANCE: "https://api.bscscan.com",
         EthereumNetwork.MATIC: "https://api.polygonscan.com",
         EthereumNetwork.OPTIMISTIC: "https://api-optimistic.etherscan.io",
+        EthereumNetwork.BOBA_RINKEBY: "https://blockexplorer.rinkeby.boba.network/api/",
+        EthereumNetwork.BOBA: "https://blockexplorer.boba.network/api/",
```
Comment (Uxio0) [inline gnosis/eth/clients/etherscan_client.py:48]: This is not etherscan, it should be removed from here

**Review by Uxio0 — CHANGES_REQUESTED** [review]

**Comment by Uxio0** [comment]
Happy to hear that you will be using Gnosis Safe on your network!!

File: gnosis/eth/clients/etherscan_client.py (hunk 1 above)
Comment (InoMurko) [reply gnosis/eth/clients/etherscan_client.py:34]: ah, I completely missed the class name.

**Comment by InoMurko** [comment]
> Happy to hear that you will be using Gnosis Safe on your network!!

Thank you @Uxio0. Are there any other places in the gnosis-py lib I need to add new chain supporting code?

**Comment by Uxio0** [comment]
You would need to add the safe contracts (v1.3.0) deployed in the networks https://github.com/gnosis/safe-deployments/tree/main/src/assets/v1.3.0 and the deployment blocks on our indexer so you can spin it up: https://github.com/gnosis/safe-transaction-service/blob/master/safe_transaction_service/history/management/commands/setup_service.py

**Comment by InoMurko** [comment]
> You would need to add the safe contracts (v1.3.0) deployed in the networks https://github.com/gnosis/safe-deployments/tree/main/src/assets/v1.3.0 and the deployment blocks on our indexer so you can spin it up: https://github.com/gnosis/safe-transaction-service/blob/master/safe_transaction_service/history/management/commands/setup_service.py

awesome, just doing that actually!

**Review by Uxio0 — APPROVED** [review]
Yes sure, forgot to approve!

**Pull request merged**